src/main/groovy/de/dkfz/roddy/execution/jobs/JobState.java

```diff
@@ -12,57 +12,43 @@
  * A generic jobState for jobs.
  */
 public enum JobState implements Serializable {
+    HOLD,
+    QUEUED,
     /**
      * BEJob is still running
      */
     RUNNING,
+    SUSPENDED,
     /**
      * BEJob has failed
      */
     FAILED,
     /**
-     * BEJob was ok
-     */
-    OK,
-    /**
      * A job where we don't know, whether it is ok or failed.
      * Has to be read out from a jobstate logfile or retrieved from the job system.
      * E.g. PBS only shows C
      */
     COMPLETED_UNKNOWN,
     /**
-     * BEJob jobState is unknown
-     */
-    UNKNOWN,
-    /**
-     * BEJob jobState is unknown because it was freshly read out from a file.
-     */
-    UNKNOWN_READOUT,
```

Comment:
* "OK" is only used in the ExecutionService
* "Unstarted"  is used -> I would keep it
* "Started" is used to check JobStatus against it in ExecutionContextReaderAndWriter, but "Started" is nowhere set
* "Unknown_Readout","Unknown_Submitted","Failed_Possible" are never used